output/v0.0.json

```diff
@@ -3615,6 +3692,23 @@
                     "payment_amount_cents": {
                         "type": "integer",
                         "readOnly": true
+                    },
+                    "metadata": {
+                        "type": "object",
+                        "readOnly": true,
+                        "description": "Additional private data represented as key-value pairs."
+                    },
+                    "reconciliation_id": {
+                        "type": "string",
+                        "readOnly": true
+                    },
+                    "payment_group_id": {
+                        "type": "string",
+                        "readOnly": true
```

Comment:
Description: Optional value that will be used to group invoices into payments. When specified, only invoices with the same value will be included in a single payment.